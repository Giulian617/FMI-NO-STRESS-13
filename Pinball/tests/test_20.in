100 18330
49 85 6
42 49 6
56 49 4
31 42 3
25 85 3
53 25 7
1 49 6
27 53 6
90 1 10
3 49 9
18 3 9
55 31 3
37 18 1
97 85 9
51 53 7
96 37 1
35 96 5
48 56 2
76 37 1
71 51 10
60 48 6
44 42 9
12 49 1
10 1 2
72 10 8
45 96 3
84 3 2
65 35 6
70 49 3
68 49 9
22 76 5
80 45 6
88 65 5
6 60 6
43 55 3
39 85 1
93 18 1
8 51 8
23 12 1
47 23 5
5 22 5
24 76 8
2 84 1
91 37 8
59 27 6
28 71 6
34 2 8
79 68 3
95 79 5
67 5 9
78 27 2
29 51 4
50 29 4
87 31 8
13 97 3
98 13 5
52 65 7
81 24 3
94 49 10
4 27 10
75 23 1
30 35 5
38 65 8
33 5 6
58 30 5
21 94 1
66 21 1
83 42 8
41 81 6
15 43 7
92 83 10
100 41 4
11 93 6
17 65 2
86 6 7
73 84 10
82 28 1
14 12 5
61 59 7
16 84 8
40 28 3
32 95 6
7 66 6
69 35 2
57 47 2
99 4 6
77 52 4
74 2 5
89 7 7
46 65 1
36 1 8
9 29 7
20 1 10
54 8 2
64 69 2
26 57 7
19 93 9
62 74 2
63 40 3
