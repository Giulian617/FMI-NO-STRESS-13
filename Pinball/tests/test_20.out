551973673
550237821
330059660
111398080
979818128
272000966
634670257
896022639
532433675
671470142
268538844
437789741
613729437
567576407
624771534
113774658
853652727
981368047
536288604
357105723
858383636
889356744
992040210
113497348
713851177
183361089
828710135
908750535
679655451
730939423
971907610
952096659
386633116
336665576
459564917
833242542
467020337
896022639
62766240
65064151
118020221
54010632
613759018
